src/components/Dashboard/TodoList.vue

```diff
@@ -1071,4 +1121,12 @@ export default {
 .todo-item-toggle h3 {
   margin-bottom: 0.25rem;
 }
+
+.bcol-error {
+  border-top: solid #a94442 3px;
```

Comment:
If this colour is something that matches one of the theme colours in the assets folder, then perhaps use of of those? I haven't seen the designs, but if it's something new/custom for the BCOL messages and we use it more than once, perhaps we could add it to the assets for re-use!

Reply:
right now, it's pretty isolated to this page and this scenario, so I don't know how much value there would be?